e_os.h

```diff
@@ -507,6 +507,7 @@ struct servent *PASCAL getservbyname(const char *, const char *);
 #    define strdup _strdup
 #   endif
 #   define unlink _unlink
+#   define fileno _fileno
```

Comment:
Thanks. Just in case for reference, what kind of overhaul. I reckon that e_os.h is used in too many places essentially unjustified. This is private and quite "heavy-weight" header and it should really be used in cases when there are strong system dependencies. Maybe more importantly, the fact that it's private should actually disqualify it for use in apps/\*, which should really set an example of using only public headers.